RELEASING.md

```diff
@@ -12,4 +12,79 @@ Create a git tag and a new release on GitHub.
 
 # Publishing the new App
```

Comment:
Add instructions to generate and check the sha256sum of all files too?

Reply:
Done